addOns/webuipoc/src/main/pocs/reactWebUI/package.json

```diff
@@ -37,6 +37,10 @@
     ]
   },
   "devDependencies": {
+    "@eslint/js": "^9.4.0",
+    "eslint": "^8.57.0",
```

Comment:
We are back to two major versions, if v8 is still a must (jsx-eslint/eslint-plugin-react#3699) we should not add v9 and keep v8.

Reply:
it comes by default while integrating Eslint, Removing it